### DISCUSSION TITLE
Lab 02.04 Task 2

### DISCUSSION THREAD

**Discussion by mariaprojas** [rramosp/ai4eng.v1#230] [02 - PYTHON]
Buenas tardes,

El código funciona bien en la prueba, pero al ejecutar el corrector sale esto:

![image](https://user-images.githubusercontent.com/113949857/221373634-2740a545-d0cb-49aa-a428-028c262d8db3.png)
![image](https://user-images.githubusercontent.com/113949857/221373639-06351361-6349-4e6f-87dd-1fe951d7829d.png)

Por favor y gracias @rramosp @imEag @raulramosudea

**Comment by mariaprojas** [comment]
Ya lo arreglé, gracias <3

**Comment by rramosp** [comment]
bien!!!

On Sat, Feb 25, 2023 at 2:38 PM María Paula Rojas Ortega <
***@***.***> wrote:

> Ya lo arreglé, gracias <3
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/230#discussioncomment-5110217>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKSXBA6GALKZZTDXD3WZJNUVANCNFSM6AAAAAAVH7ZLJU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
